app/main.py

```diff
@@ -1,7 +1,22 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+
+def get_weather(api_key: str) -> str:
+    url = f"http://api.weatherapi.com/v1/current.json?key={api_key}&q=Paris"
```

Comment:
it is bad practice to pass url params with f-string, use `params` param in requests.get method instead